Modules/Core/Common/include/itkOptimizerParameters.hxx

```diff
@@ -23,70 +23,51 @@
 
 namespace itk
 {
-/** Default contstructor */
+/** Default constructor */
 template <typename TParametersValueType>
-OptimizerParameters<TParametersValueType>::OptimizerParameters()
-  : Array<TParametersValueType>()
-  , m_Helper(nullptr)
-{
-  this->Initialize();
-}
+OptimizerParameters<TParametersValueType>::OptimizerParameters() = default;
```

Comment:
Is there a reason for not moving this declaration to .h file now that it is trivial?